debian/rules

```diff
@@ -35,7 +35,7 @@ else
 endif
 
 %:
-	dh $@ --buildsystem=cmake --with javahelper,python2,python3,systemd --parallel
+	dh $@ --buildsystem=cmake --with javahelper,python3,systemd --parallel
```

Comment:
might want to update the commit message

- add the "debian" prefix
- update the commit message to note that "remove python2 from dh options"
- better off squashing d699e81da58dfddac56d2f6ca4d473b47d973abb into this one